src/archanaNJun20/BookMeetingRoom.java

```diff
@@ -0,0 +1,105 @@
+package archanaNJun20;
+
+public class BookMeetingRoom {
+	
+	double nalandaHours,takshashilaHours,agraFortHours,pratapGadhHours;
+	boolean nalandaFlag,takshashilaFlag,agraFortFlag,pratapGadhFlag;
+	enum MEETINGROOM{Nalanda,Takshashila,AgraFort,PratapGadh};
+	
+	void bookMeetingRoom(){
+		if(!nalandaFlag){
+			System.out.println("Nalanda meeting room  is  booked for 1hr.");
+			nalandaHours=1;
+			nalandaFlag=true;
+		}
+		else{
+			System.out.println("Nalanda meeting  room  is already booked.");
+		}
+	}
+	void bookMeetingRoom(MEETINGROOM meetingRoom){
+		bookMeetingRoom(meetingRoom, 1);
+	}
+
+	void bookMeetingRoom(MEETINGROOM meetingRoom,int hours){
+		if(meetingRoom==MEETINGROOM.Nalanda){
+			if(nalandaFlag==false){
+				System.out.println(meetingRoom+" meeting room is booked for "+hours+"hr");
+				nalandaHours=hours;
+				nalandaFlag=true;
+			}
+			else 
+				System.out.println("Nalanda meeting room  is already booked.");
+		}else if(meetingRoom==MEETINGROOM.Takshashila) {
+			System.out.println(meetingRoom+" meeting room is booked for "+hours+"hr");
+			if(takshashilaFlag==false){
+				takshashilaHours=hours;
+				takshashilaFlag=true;
+			}
+			else
+				System.out.println(meetingRoom+" meeting room  is  already booked.");
+		}
+		else if(meetingRoom==MEETINGROOM.AgraFort) {
+			if(agraFortFlag==false){
+				System.out.println(meetingRoom+" meeting room is booked for "+hours+"hr");
+				agraFortHours=hours;
+				agraFortFlag=true;
+			}
+			else
+				System.out.println("AgraFort meeting room  is already booked.");
+		}
+		else if(meetingRoom==MEETINGROOM.PratapGadh) {
+			if(pratapGadhFlag==false){
+				System.out.println(meetingRoom+" meeting room is booked for "+hours+"hr");
+				pratapGadhHours=hours;
+				pratapGadhFlag=true;
+			}
+			else
+				System.out.println("PratapGadh meeting room  is already booked.");
+		}
```

Comment:
no need to write multiple else condition as you are already using if...else if ladder.
you can write one else condition at end.

Reply:
Code is updated with all review comments,